mcstatus/protocol/connection.py

```diff
@@ -236,7 +246,7 @@ def __repr__(self) -> str:
     @staticmethod
     def _unpack(format_: str, data: bytes) -> int:
         """Unpack data as bytes with format in big-endian."""
-        return struct.unpack(">" + format_, bytes(data))[0]
+        return cast(int, struct.unpack(">" + format_, bytes(data))[0])
```

Comment:
You don't need to cast anything here.

Reply:
Duplicate to https://github.com/py-mine/mcstatus/pull/555#discussion_r1209516063.